_includes/items.html

```diff
@@ -3,13 +3,6 @@
 {% for post in posts reversed %}
   {% unless post.tags contains 'switch' %}
     {% unless post.tags contains 'stick_to_windows' %}
-      <a href="{{ post.url }}">
```

Comment:
Why this change? This removes most of what is on the home page. Is this intentional?

Reply:
Okay I think I made a change to resolve the problem you saw. Could you update your pull request to revert this file but leave the other changes? Thanks!

Reply:
Sorry, I’m new to GitHub, how I do I revert this file on desktop?

Reply:
Never mind, I figured it out, I think the pull request is good now.